### ISSUE TITLE
Non-fatal java.lang.NoSuchMethodError: com.sun.jna.LastErrorException using devservices

### ISSUE THREAD

**Issue by laurentperez** [quarkiverse/quarkus-jooq#18]
Hi

Quarkus 1.13.1, devservices enabled, JVM mode (not GraalVM, not native), quarkus-jooq 0.1.0 (yes, it's fresh ;) ), dialect POSTGRES (not H2)

`./gradlew quarkusDev` throws an Exception starting the devservices Testcontainer (thru Docker), but it still deploys and run
```
Exception in thread "docker-java-stream-1001451016" java.lang.NoSuchMethodError: com.sun.jna.LastErrorException: method <init>(Ljava/lang/String;)V not found
        at org.testcontainers.shaded.com.github.dockerjava.okhttp.UnixDomainSocket.recv(Native Method)
        at org.testcontainers.shaded.com.github.dockerjava.okhttp.UnixDomainSocket$UnixSocketInputStream.read(UnixDomainSocket.java:246)
        at java.base/java.io.FilterInputStream.read(FilterInputStream.java:133)
        at org.testcontainers.shaded.com.github.dockerjava.okhttp.UnixSocketFactory$1$1.read(UnixSocketFactory.java:45)
        at org.testcontainers.shaded.okio.Okio$2.read(Okio.java:140)
```

I guess it's somehow related to the Redhat UBI microdnf image used at https://github.com/quarkusio/quarkus-quickstarts/blob/main/getting-started-async/src/main/docker/Dockerfile.jvm since it's openjdk and not Sun.

**Comment by angrymango** [comment]
Hi @laurentperez thanks for the bug report. I think the issue is related to the inclusion of a stubbed version of LastErrorException which was included to enable native compilation to work. See here the related issue here: https://github.com/quarkusio/quarkus/issues/6327

I've done some initial testing and it seems like this is no longer an issue and the stub can probably be removed. I'll push an update when I've completed testing.

**Comment by angrymango** [comment]
LastErrorException stub was removed in https://github.com/quarkiverse/quarkus-jooq/commit/8a82d15159e3912dd1005d6410d9c5c08f3565e5

**Comment by angrymango** [comment]
Fixed in release 0.2.0 - https://github.com/quarkiverse/quarkus-jooq/releases/tag/0.2.0
